Dockerfile.base

```diff
@@ -0,0 +1,38 @@
+FROM python:3.6-alpine
+LABEL maintainer "EGA System Developers"
+
+RUN apk add --no-cache \
+    libressl libressl-dev make gcc musl-dev libffi-dev postgresql-libs postgresql-dev git
```

Comment:
This line fails on the 3.6 base image, libressl-dev needs to be replaced with openssl-dev

Reply:
Ok.
That's not the case for me, but I can double-check that.

Reply:
there is an issue in `alpine:3.9` vs `apline3.8` if you are getting:
```
ERROR: unsatisfiable constraints:
  openssl-dev-1.1.1a-r1:
    conflicts: libressl-dev-2.7.4-r2[pc:libcrypto=1.1.1a]
               libressl-dev-2.7.4-r2[pc:libssl=1.1.1a]
               libressl-dev-2.7.4-r2[pc:openssl=1.1.1a]
    satisfies: curl-dev-7.63.0-r0[openssl-dev]
               libssh2-dev-1.8.0-r4[pc:libcrypto]
               libssh2-dev-1.8.0-r4[pc:libssl]
  libressl-dev-2.7.4-r2:
    conflicts: openssl-dev-1.1.1a-r1[pc:libcrypto=2.7.4]
               openssl-dev-1.1.1a-r1[pc:libssl=2.7.4]
               openssl-dev-1.1.1a-r1[pc:openssl=2.7.4]
    satisfies: world[libressl-dev]
               libssh2-dev-1.8.0-r4[pc:libcrypto]
               libssh2-dev-1.8.0-r4[pc:libssl]
```
I suggest using `alpine3.8`

Reply:
Correct we need the from line to read: `FROM python:3.6-alpine3.8`